open-api/rest-catalog-open-api.yaml

```diff
@@ -3324,6 +3324,280 @@ components:
           type: integer
           format: int64
 
+    BooleanTypeValue:
+      type: boolean
+      example: true
+
+    IntegerTypeValue:
+      type: integer
+      example: 42
+
+    LongTypeValue:
+      type: integer
+      format: int64
+      example: 9223372036854775807
+
+    FloatTypeValue:
+      type: number
+      format: float
+      example: 3.14
+
+    DoubleTypeValue:
+      type: number
+      format: double
+      example: 123.456
+
+    DecimalTypeValue:
+      type: string
+      description:
+        "Decimal type values are serialized as strings. Decimals with a positive scale serialize as numeric plain 
+        text, while decimals with a negative scale use scientific notation and the exponent will be equal to 
+        the negated scale"
+      example:
```

Comment:
I checked that the way we distinguish one example of a struct vs multiple examples is that you say `examples` here instead of `example`: https://swagger.io/docs/specification/adding-examples/

Could you check all related places and fix it?

Reply:
the `examples` key doesn't apply to objects or properties. This can only be applied to parameters, requests, and response models

Reply:
If we don't want to represent this as a map of examples, I can either make this a three different decimal types or add response examples once this model is ingested in either scan planning or append API.

Reply:
I see. In that case I suggest we only put 1 example in `example`, and the other examples in the description section.